resources/plugins/browser/index.js

```diff
@@ -0,0 +1,35 @@
+const IS_URL = /(?:https?:\/\/)?([\w\d-]+(?:\.[\w]+)+$)/;
+
+/**
+ * Returns the normalized URL (with protocol scheme)
+ *
+ * @param {String} url
+ * @return {String}
+ */
+const normalize = (url) => {
+  console.log('test', url);
+  if (/^https?:\/\//.test(url)) {
+    return url;
+  }
+  return `https://${url}`;
+};
+
+module.exports = {
+  action: 'openurl',
+  query: (query) => {
+    const items = [];
+    const urls = IS_URL.exec(query);
+    if (urls) {
+      items.push({
+        title: `Open ${query} in the browser.`,
+        browser: 'Open in browser',
+        arg: normalize(query),
+        icon: {
+          type: 'text',
+          letter: urls[1].substr(0, 1).toUpperCase(),
```

Comment:
if you add a path after the domain, the letter will the first letter of the path, not one from the domain.
`urls = is_url.exec('http://www.google.com/test.htm');`
Letter: 'T'

Is that the idea?